### PR TITLE
Initial Commit

### LINKED ISSUE #3: Scaffold home page

### PR DESCRIPTION
**Pull request by TeoMastro**
Closes #3 
# Pull Request Template

Clean up this template and provide relevant information about your pull request!

## All Submissions

* [ ] Have you followed the guidelines in our [Contributing](../.github/CONTRIBUTING.md) document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Does your pull request target an open issue?

### New Feature Submissions

* [ ] Does your submission pass tests?
* [ ] Have you lint your code locally prior to submission?

### Changes to Core Features

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: src/Ieemdb.Web/ClientApp/src/App.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
 import React, { Component } from 'react';
-import { Route } from 'react-router';
-import { Layout } from './components/Layout';
-import { Home } from './components/Home';
-import { FetchData } from './components/FetchData';
-import { Counter } from './components/Counter';
-import AuthorizeRoute from './components/api-authorization/AuthorizeRoute';
-import ApiAuthorizationRoutes from './components/api-authorization/ApiAuthorizationRoutes';
-import { ApplicationPaths } from './components/api-authorization/ApiAuthorizationConstants';
```
Comment (esentis) [inline src/Ieemdb.Web/ClientApp/src/App.js]: Authorization components are required for user validation

File: src/Ieemdb.Web/ClientApp/src/App.js [hunk 2]
```diff
@@ -15,12 +11,12 @@ export default class App extends Component {
 
   render () {
     return (
-      <Layout>
-        <Route exact path='/' component={Home} />
-        <Route path='/counter' component={Counter} />
-        <AuthorizeRoute path='/fetch-data' component={FetchData} />
-        <Route path={ApplicationPaths.ApiAuthorizationPrefix} component={ApiAuthorizationRoutes} />
```
Comment (esentis) [inline src/Ieemdb.Web/ClientApp/src/App.js:22]: Authorization components are required for user validation

**Review by esentis — CHANGES_REQUESTED** [review]
Δεν δουλεύει πια το user authorization/authentication με τα components που αφαίρεσες

**Comment by esentis** [comment]
Διαβάζουμε το description που συμπληρώνουμε 🙏

**Review by esentis — APPROVED** [review]
Seems ok !

**Review by KostisGrf — COMMENTED** [review]
Seems ok

**Review by GavriilJim — APPROVED** [review]
Seems ok

**Review by esentis — APPROVED** [review]

**Review by KostisGrf — APPROVED** [review]

**Pull request merged**